template/web/wp-content/themes/<%= slug %>/app/Managers/TinyMCEManager.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/**
+ * Bootstraps TinyMCE related functions.
+ *
+ * @package Studiometa
+ */
+
+namespace Studiometa\Managers;
+
+use Studiometa\Managers\ManagerInterface;
+
+/** Class */
+class TinyMCEManager implements ManagerInterface {
+	// phpcs:ignore Generic.Commenting.DocComment.MissingShort
+	/**
+	 * @inheritDoc
+	 */
+	public function run() {
+		add_filter( 'tiny_mce_before_init', array( $this, 'set_editor_config' ) );
+		add_action( 'admin_print_styles', array( $this, 'add_editor_stylesheet' ), 11 );
+		add_filter( 'mce_buttons', array( $this, 'add_editor_buttons' ) );
+	}
+
+	/**
+	 * Add a buttons to the TinyMCE editor
+	 *
+	 * @param array $buttons Buttons list.
+	 */
+	public function add_editor_buttons( $buttons ) {
+		array_unshift( $buttons, 'styleselect' );
+
+		return $buttons;
+	}
+
+	/**
+	 * Set TinyMCE Editor config
+	 * - add custom style formats dropdown
+	 * - add custom colors
+	 *
+	 * @param array $config Configuration.
+	 * @return array
+	 */
+	public function set_editor_config( $config ) {
+		$style_formats = array(
+			array(
+				'title'    => __( 'Large title' ),
+				'selector' => 'p, h1, h2, h3, h4, h5, h6',
+				'classes'  => 'type-h1',
+			),
+			array(
+				'title'    => __( 'Button' ),
+				'selector' => 'a',
+				'classes'  => 'btn',
+			),
+		);
+
+		$colors = array(
+			'000000', // Hexadecimal color value.
+			'Black', // Color name.
+			'ffffff',
+			'White',
+		);
+
+		// Add custom style_formats to TinyMCE.
+		$config['style_formats'] = wp_json_encode( $style_formats );
+
+		// Add custom colors to TinyMCE.
+		$config['textcolor_map'] = wp_json_encode( $colors );
```

Comment:
**question:** Would it be useful to import colors from the Tailwind configuration?

Reply:
It could be. It depends if we want to let the user have access to all the config colors, or limit the number of colors they can use